gradle/libs.versions.toml

```diff
@@ -276,6 +276,7 @@ testRules = { module = "androidx.test:rules", version.ref = "androidxTestRules"
 testRunner = { module = "androidx.test:runner", version.ref = "androidxTestRunner" }
 testUiautomator = { module = "androidx.test.uiautomator:uiautomator", version = "2.2.0" }
 truth = { module = "com.google.truth:truth", version = "1.0.1" }
+turbine = { module = "app.cash.turbine:turbine", version = "1.1.0" }
```

Comment:
I'd avoid adding a new library to monorepo just for one test

Reply:
My vote is to postpone this decision and do this PR without it.
@igordmn wdyt?

Reply:
If we take into account my comment about `currentStateFlow`, this library isn't needed, so we can postpone the decision.

But if it would needed, then I would include it if someone find it very useful, and others don't have strong arguments against it. We have a policy - 2 reviewers for such changes.

Personally, I don't have strong arguments against it, nor strong arguments for it. Maybe I have slightly arguments for it, after reading the description. Small stable libraries usually prove itself useful.

One potential argument against it - security reasons (a library can tamper a developer machine). But it less relevant with established vendors as cashapp.

Reply:
Removed it for now